app/javascript/react/screens/App/Plan/components/PlanRequestDetailList/PlanRequestDetailList.js

```diff
@@ -247,7 +247,7 @@ class PlanRequestDetailList extends React.Component {
   overlayTriggerClick = task => {
     if (task.options.playbooks) {
       const playbookStatuses = task.options.playbooks;
-      let runningPlaybook = null;
+      let runningPlaybook;
```

Comment:
We set the default value to `null` here [(review comment)](https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/442#discussion_r204047438)